### PR TITLE
test: update jest and testing library

### PR DESCRIPTION
**Pull request by areisle**
updating jest etc. before the swap to typescript

### CODE REVIEW

File: src/views/AdvancedSearchView/__tests__/index.test.js [hunk 1]
```diff
@@ -58,45 +59,42 @@ describe('AdvancedSearchView', () => {
...
-  let getByTestId;
-  let getByText;
-
-  beforeEach(() => {
-    ({
-      getByTestId, getByText,
-    } = render(
+  test('search button fires correctly', () => {
+    render(
       <AdvancedSearchView
```
Comment (creisle) [inline src/views/AdvancedSearchView/__tests__/index.test.js:64]: why did you move the render out of the beforeEach? (seems like the tests still all render the same)

Comment (areisle) [reply src/views/AdvancedSearchView/__tests__/index.test.js:64]: I guess it just didn't seem necessary. it only saves a few lines of code and tests are easier to read and debug when all their code is within the test block

**Review by creisle — APPROVED** [review]
